### ISSUE TITLE
[Feat]: Google OAuth

### ISSUE THREAD

**Issue by AbdulHameedi** [GitHub-Campus-Experts-Unilag/DevzenFrontend#34]
Can I be assigned @shegz101

**Comment by shegz101** [comment]
Please explain better on what you want to do. How do you intend on going about this?

**Comment by AbdulHameedi** [comment]
For the login Auth, the user should be able to continue with Google to login

**Comment by shegz101** [comment]
> For the login Auth, the user should be able to continue with Google to login

Yes, I know. I want to know if you intend to use Saas (firebase and the rest like that) or the backend API.

**Comment by AbdulHameedi** [comment]
I created another issue already, can this one be closed please
Or if it will be done by someone else

**Comment by Alfred-Emmanuel** [comment]
Could I be assigned to this?

**Comment by tdadadavid** [comment]
Pick it up, and I will assign you once I am on my laptop

**Comment by Alfred-Emmanuel** [comment]
Alright, on it
